src/main/java/com/philips/research/spdxbuilder/persistence/spdx/SpdxWriter.java

```diff
@@ -112,10 +112,10 @@ private void writePackage(TagValueDocument doc, Package pkg, BillOfMaterials bom
         }
         doc.addValue("PackageHomePage", pkg.getHomePage());
         doc.addValue("PackageLicenseConcluded", pkg.getConcludedLicense());
-        if (pkg.getConcludedLicense().isEmpty()) {
-            System.err.println("WARNING: No concluded license for package " + pkg);
-        }
         doc.addValue("PackageLicenseDeclared", pkg.getDeclaredLicense());
+        if (pkg.getDeclaredLicense().isEmpty() && pkg.getConcludedLicense().isEmpty()) {
```

Comment:
Do we want to see a difference in error message depending on missing Declared or Concluded licenses?

Reply:
The message is intended to signal that this packages does not have any license: The behaviour of SPDX readers should be to prefer the (optional) "concluded" license or else fall back to the "declared" license.